### DISCUSSION TITLE
How many 🏊 strokes are in the FINA?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#949] [Galaxy Trivia] [answered]
**Bonus:** Describe the basic technique for each.

**Comment by peterbaccaro** [comment, marked as answer]
The Fédération Internationale de Natation (FINA) which is the international governing body for swimming recognizes **four different swimming strokes** in competitive swimming. These strokes are:

- **Freestyle (also known as front crawl):** In this stroke, swimmers can use any technique they prefer, as long as it doesn't violate any other swimming rules. It is commonly characterized by the alternate arm overhand motion and the flutter kick.

- **Backstroke:** Swimmers in this stroke swim on their back throughout the race. They use an alternating motion of the arms combined with a flutter kick. The swimmer must remain on their back during the entire race, except during the turn, where a brief rotation to the stomach is allowed.

- **Breaststroke:** This stroke involves a simultaneous and symmetrical movement of the arms along with a frog-like kick. Swimmers must touch the wall with both hands simultaneously on each turn and at the finish.

- **Butterfly:** In the butterfly stroke, swimmers use a simultaneous movement of both arms over the water combined with an undulating dolphin kick. Both arms must be brought forward together, and the legs must remain together and move up and down simultaneously.
